src/mame/video/dassault.cpp

```diff
@@ -54,7 +56,12 @@ void dassault_state::mixdassaultlayer(bitmap_rgb32 &bitmap, bitmap_ind16* sprite
 
 				if (alpha!=0xff)
 				{
-					if (pix&0x600)
+					if (pix&0x400) // todo
```

Comment:
Please use uppercase `TODO` so editors like vim will highlight it and source analytics tools will recognise it.